cli/src/commands/deploy_config.py

```diff
@@ -0,0 +1,22 @@
+from .abstract_command import AbstractCommand
+
+
+class DeployConfig(AbstractCommand):
+    def get_name(self):
+        return "deploy"
+
+    def get_description(self):
+        return "Deploy a config"
+
+    def get_options(self):
+        return [{"name": "index_name",
+                 "description": "name of the config to deploy"}]
+
+    def run(self, args):
+        if len(args) <= 0:
+            print(
+                'You can only deploy one config at a time, index_name missing. Aborting')
```

Comment:
Reformulate: the message "You can only deploy one config at a time" looks now inappropriate.

Possibly go to just "Usage: deploy_config INDEX-NAME" (but I don't know what are the form of the messages of other commands)